test/rubygems/test_gem_gemcutter_utilities.rb

```diff
@@ -230,10 +230,33 @@ def test_sign_in_with_incorrect_otp_code
     assert_equal "111111", @fetcher.last_request["OTP"]
   end
 
-  def util_sign_in(response, host = nil, args = [], extra_input = "")
-    email            = "you@example.com"
-    password         = "secret"
-    profile_response = HTTPResponseFactory.create(body: "mfa: disabled\n", code: 200, msg: "OK")
+  def test_sign_in_with_webauthn_otp
+    webauthn_verification_url = "rubygems.org/api/v1/webauthn_verification/odow34b93t6aPCdY"
+    api_key       = "a5fdbb6ba150cbb83aad2bb2fede64cf040453903"
```

Comment:
Nit: `api_key` and `response_fail` are used in the opposite order in lines 241 and 243.